### PR TITLE
VTCCA-4929 updated help guidance on check/challenge tabs

### PR DESCRIPTION
**Pull request by zachhall1234**

### CODE REVIEW

File: app/views/dvr/challengeCasesDetails.scala.html [hunk 1]
```diff
@@ -78,9 +78,36 @@
...
+    </ul>
+
+    <p>@Html(messages("challengeRules.rules.paper", config.businessRatesValuationFrontendUrl(s"property-link/valuations/startChallenge?backLinkUrl=$returnUrl")))</p>
+}
+
 
 <h2 class="govuk-heading-l">@messages("property.details.div.submitted.challenge.cases")</h2>
 
+@govukDetails(Details(
+    id = Some("whenYouMayChallengeDetails"),
```
Comment (DanSmith98) [inline app/views/dvr/challengeCasesDetails.scala.html:106]: We usually format ids like `when-you-may-challenge-details`

**Comment by DanSmith98** [comment]
LGTM :+1: (one tiny comment)

File: app/views/dvr/challengeCasesDetails.scala.html (hunk 1 above)
Comment (zachhall1234) [reply app/views/dvr/challengeCasesDetails.scala.html]: Thanks @DanSmith98, updated this & the existing challenge guidance component in checkCasesDetails.scala.html

**Comment by habeeb01** [comment]
LGTM 👍

**Pull request merged**